### PR TITLE
[CHORE] Add export records option

### PR DESCRIPTION
**Pull request by Ballo1988**
- Added an option to export records in csv format

image
image

### CODE REVIEW

File: app/controllers/patients_controller.rb [hunk 1]
```diff
@@ -66,6 +68,18 @@ def destroy
     redirect_to patients_path
   end
 
+  def export
```
Comment (Diezagu) [inline app/controllers/patients_controller.rb]: I don't know if this is correct, but I remember Urbi telling me that in the controller, there should be nothing but CRUD functionality.
 
 @itsmeurbi help us here haha

**Review by Diezagu — CHANGES_REQUESTED** [review]

**Comment by Diezagu** [comment]
Additionally you have broken tests

File: app/controllers/patients_controller.rb (hunk 1 above)
Comment (itsmeurbi) [reply app/controllers/patients_controller.rb]: Not necessarily is wrong, but in terms of Rails it is better to tight to the CRUD actions
Always try to treat things as resources.

@Ballo1988 in this case, exporting patients is nothing more than requesting a list of patients in csv format; that is an index action

Same for the consultants. You're requesting a list of them in csv format

Please refer to https://apidock.com/rails/v7.1.3.2/ActionController/MimeResponds/respond_to

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   resources :patients do
+    collection do
+      get :export, defaults: { format: :zip }
```
Comment (itsmeurbi) [inline config/routes.rb]: You don't need this anymore

**Review by itsmeurbi — APPROVED** [review]

**Pull request merged**